content/docs/command-reference/pull.md

```diff
@@ -13,8 +14,8 @@ usage: dvc pull [-h] [-q | -v] [-j <number>]
                 [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these DVC-files. Using -R,
-                 directories to search DVC-files in can also be given.
+  targets        Limit command scope to these `.dvc` files. Using -R,
+                 directories to search `.dvc` files in can also be given.
```

Comment:
```suggestion
  targets        Limit command scope to these stages or `.dvc` files.
                 Using -R, directories can also be given, to search
                 `.dvc` files in.
```

Reply:
I hope directly committing this is okay?

Reply:
Yes, as long as it doesn't break the formatting. My suggestions in GH are just eye-balled sometimes.